### PR TITLE
[no ticket][risk=no] Puppeteer refactor owner-copy-to-workspace test

### PR DESCRIPTION
**Pull request by aweng98**
Refactor test to reduces playback time by half and use one less new workspace.

### CODE REVIEW

File: e2e/tests/notebook/owner-copy-to-workspace.spec.ts [hunk 1]
```diff
@@ -18,64 +19,87 @@ jest.setTimeout(20 * 60 * 1000);
...
+   const workspace = await createWorkspace(page, cdrVersion);
+   return workspace.getWorkspaceName();
+}
 
-  beforeEach(async () => {
-    await signIn(page);
-  });
+// Reuse same source workspace for all tests in this file, in order to reduce test playback time.
+// Workspace to be created in first test. If create failed in first test, next test will try create it.
+let srcWorkspace: string;
```
Comment (jmthibault79) [inline e2e/tests/notebook/owner-copy-to-workspace.spec.ts:85]: Let's call this `defaultCdrWorkspace`

**Review by jmthibault79 — APPROVED** [review]
Nice change, thanks!

Is there any way we can delete the sourceWorkspace after all tests in this file have run?

File: e2e/tests/notebook/owner-copy-to-workspace.spec.ts (hunk 1 above)
Comment (aweng98) [reply e2e/tests/notebook/owner-copy-to-workspace.spec.ts]: Done

**Review by aweng98 — COMMENTED** [review]
To delete the sourceWorkspace, we can add code at the end of the last test, in `test(), async() =>{ }` block. I'll save this for nex time.
thank you.

**Pull request merged**